### ISSUE TITLE
COMMUNICATION_ERROR

### ISSUE THREAD

**Issue by maciekish** [scurb/NeatoBinding#5]
Hi, I just got a Botvac D5 Connected and updated its firmware to 4.x. iOS app says Read to Clean. But the binding is spamming the console with these errors. I added it though Paper UI, Inbox, Search. Any ideas please?

    2018-01-27 10:12:18.526 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from OFFLINE (COMMUNICATION_ERROR) to ONLINE
    2018-01-27 10:12:18.527 [ERROR] [b.binding.neato.handler.NeatoHandler] - Error when refreshing state. Error: null
    2018-01-27 10:12:18.527 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from ONLINE to OFFLINE (COMMUNICATION_ERROR)
    2018-01-27 10:12:19.195 [INFO ] [b.binding.neato.handler.NeatoHandler] - Will get STATE for Robot Eryk
    2018-01-27 10:12:19.511 [INFO ] [b.binding.neato.handler.NeatoHandler] - Successfully got and parsed new state for Eryk
    2018-01-27 10:12:19.512 [ERROR] [b.binding.neato.handler.NeatoHandler] - Error when refreshing state. Error: null
    2018-01-27 10:12:19.512 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from OFFLINE (COMMUNICATION_ERROR) to ONLINE
    2018-01-27 10:12:19.513 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from ONLINE to OFFLINE (COMMUNICATION_ERROR)
    2018-01-27 10:12:20.195 [INFO ] [b.binding.neato.handler.NeatoHandler] - Will get STATE for Robot Eryk
    2018-01-27 10:12:20.725 [INFO ] [b.binding.neato.handler.NeatoHandler] - Successfully got and parsed new state for Eryk
    2018-01-27 10:12:20.725 [ERROR] [b.binding.neato.handler.NeatoHandler] - Error when refreshing state. Error: null
    2018-01-27 10:12:20.726 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from OFFLINE (COMMUNICATION_ERROR) to ONLINE
    2018-01-27 10:12:20.726 [hingStatusInfoChangedEvent] - 'neato:vacuumcleaner:OPS32517-7C38********' changed from ONLINE to OFFLINE (COMMUNICATION_ERROR)
    2018-01-27 10:12:21.195 [INFO ] [b.binding.neato.handler.NeatoHandler] - Will get STATE for Robot Eryk
    2018-01-27 10:12:21.516 [INFO ] [b.binding.neato.handler.NeatoHandler] - Successfully got and parsed new state for Eryk
    2018-01-27 10:12:21.516 [ERROR] [b.binding.neato.handler.NeatoHandler] - Error when refreshing state. Error: null

**Comment by HerzScheisse-zz** [comment]
yep... can confirm this... also on newest D5 connected firmware... and have the same error

any help to solve this?

**Comment by maciekish** [comment]
A fix is available here https://community.openhab.org/t/neato-botvac-connected-binding/16159/183?u=maciekish
